app/containers/ImageSkeletonPlaceholder/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import FastImage, { FastImageProps } from 'react-native-fast-image';
+import SkeletonPlaceholder from 'react-native-skeleton-placeholder';
+
+const ImageSkeletonPlaceholder = ({ style, ...otherProps }: FastImageProps): React.ReactElement => {
+	const [isLoading, setLoading] = useState(true);
+
+	const handleLoadEnd = () => {
+		setLoading(false);
+	};
+
+	// We inject relevant properties of the `style` passed to the FastImage
+	// so as to avoid a "jump" when the image finishes loading:
+	const flattenedStyle = StyleSheet.flatten(style); // may be undefined
+
+	const passedHeight = flattenedStyle?.height;
+	const passedMinHeight = flattenedStyle?.minHeight;
+
+	let placeholderItemHeight: string | number | undefined;
+	let placeholderItemMinHeight: string | number | undefined;
+
+	if (passedHeight) {
+		placeholderItemHeight = passedHeight;
+	} else if (passedMinHeight) {
+		placeholderItemMinHeight = passedMinHeight;
+	} else {
+		placeholderItemHeight = '100%'; // by default
+	}
+
+	const skeletonPlaceholder = (
+		<View testID='image-skeleton-placeholder-skeleton-placeholder'>
+			<SkeletonPlaceholder>
+				<SkeletonPlaceholder.Item
+					height={placeholderItemHeight}
+					width={flattenedStyle?.width ?? '100%'}
+					minHeight={placeholderItemMinHeight}
+					borderRadius={flattenedStyle?.borderRadius}
+				/>
+			</SkeletonPlaceholder>
+		</View>
+	);
+
+	return (
+		<View testID='image-skeleton-placeholder'>
+			{isLoading && skeletonPlaceholder}
+			{/* Hide <FastImage /> while it is loading: */}
+			<FastImage style={[{ display: isLoading ? 'none' : 'flex' }, style]} onLoadEnd={handleLoadEnd} {...otherProps} />
```

Comment:
More of a comment than a requested change, but we need to check `onLoadError` as well to stop showing the placeholder forever.
We can do it on a different PR though, since we don't actually have anything in mind right now.

Reply:
Good point. I can add a very basic error message display in this PR. What do you think of the design attached?

![Capture d’écran 2023-03-14 à 09 28 13](https://user-images.githubusercontent.com/79283128/224940939-30525cdb-41e7-4d9a-ad29-aec35c1f9689.png)

Reply:
We can't do it that way, because depending on the workspace configuration we add userId and token to the url.
Check ShareView on iOS.
Sometimes we need to display a `warning` icon with a message.
Maybe we could do that icon with a simple `Can't display image`.
Just remember to style this text, so it uses font and colors like the rest of the app and we're fine.
I can talk to design team to polish it in the future, if needed.
Thanks!

Reply:
@floels

Reply:
@diegolmello I added this behavior to the code and tests and I included a screenshot of the resulting design in the PR description.

Let me know what you think!

Reply:
lgtm!